server/graphql-schema.js

```diff
@@ -425,12 +425,7 @@ const graphqlSchema = gql`
         The ATs the test was written to expect.
         """
         ats: [At]!
-        # TODO: consider moving to the Scenario type if we support a single test
-        # applying to multiple AT modes.
-        """
-        The AT mode the test was written to expect.
-        """
-        atMode: AtMode
+
```

Comment:
Would you mind also removing the extra blank line, just to keep it consistent with the rest of the file.

Reply:
There are other references to AtMode in this file. Can you remove them as well?

```
        # TODO: reenable when this data is properly flowing into the system
        # """
        # The categories of generalized AT modes the AT supports.
        # """
        # modes: [AtMode]!
```

```
    """
    Some ATs like JAWS have modes - for example, normally pushing "h" would
    navigate to the next header, but when you are interacting with a text input,
    typing "h" would insert the letter "h" into that input. Our tests are aware
    of these modes.
    """
    enum AtMode {
        """
        JAWS "browse" mode or NVDA "browse" mode.
        """
        READING
        """
        JAWS "forms" mode, NVDA "focus" mode, or VoiceOver's one-and-only mode.
        """
        INTERACTION
    }
```